components/automate-chef-io/data/docs/api-static/02-meta-description.yaml

```diff
@@ -3,12 +3,46 @@ info:
     # Authentication
     The standard way of authenticating for the Automate API is to generate an
     API token and pass that with your API requests with the `api-token` header.
-    An example request using curl looks like this:
+
+    An administrative token can be created and set to a variable using the following CLI helper:
```

Comment:
i wonder if we should say `admin token` instead of `administrative token`... since i think thats how we refer to it elsewhere.  just wondering, no strong opinion about it.